operation/fhir-operation-bulkdata/src/docs/Configuration.md

```diff
@@ -0,0 +1,79 @@
+The following is a brief template of the V2 Configuration. 
```

Comment:
I think its the first time I've seen us put info under `src/docs`.  Do we want to adopt this pattern going forward?
If not, maybe just stuff it into the fhir-operation-bulkdata README?

Reply:
moved/changed